drivers/sensor/sensirion/scd4x/scd4x.c

```diff
@@ -148,16 +207,14 @@ static int scd4x_init(const struct device *dev)
         return -ENODEV;
     }
 
-    // Stop periodic measurement
     ret = scd4x_send_command(dev, SCD4X_CMD_STOP_PERIODIC_MEASUREMENT);
     if (ret < 0) {
         LOG_ERR("Failed to stop periodic measurement");
         return ret;
     }
 
-    k_sleep(K_MSEC(500)); // Wait for stop
+    k_sleep(K_MSEC(500));
 
```

Comment:
Comment format will be addressed in an other issue.

```suggestion
    // Start periodic measurement
```